csharp-basics/exercises/Arrays/Arrays/Exercise8/Program.cs

```diff
@@ -0,0 +1,244 @@
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 4:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("   O   |   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 5:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("   O   |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 6:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("   |   |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 7:
+                {
+                    Console.WriteLine("   +---+   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("       |   ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+                case 8:
+                {
+                    Console.WriteLine("           ");
+                    Console.WriteLine("           ");
+                    Console.WriteLine("           ");
+                    Console.WriteLine("           ");
+                    Console.WriteLine("           ");
+                    Console.WriteLine("           ");
+                    Console.WriteLine("===========");
+                    break;
+                }
+            }
+        }
+
+        static char CheckLetter(int index)
+        {
+            return hiddenWord[index];
+        }
+
+        static bool GameWon()
+        {
+            if (hiddenWord.Contains('_'))
+            {
+                return false;
+            }
+            
+            Console.WriteLine("You won!");
+            return true;
+        }
+
+        static bool GameLost()
+        {
+            if (guessesRemaining > 0)
```

Comment:
if (guessesRemaining < 0)
            {
                Console.WriteLine("You lost!");
            }
return guessesRemaining < 0;